src/script/api/script_company.hpp

```diff
@@ -46,6 +48,52 @@ class ScriptCompany : public ScriptObject {
 		GENDER_INVALID = -1, ///< An invalid gender.
 	};
 
+	enum LiveryScheme {
+		LS_DEFAULT,
```

Comment:
documentation? :>

Real shame these enums have to be duplicated like this...

Reply:
IMHO this PR would be good to get in for 1.9.0, any chance of this small point being fixed?